nuget/lib/dependabot/nuget/file_updater.rb

```diff
@@ -79,6 +79,11 @@ def updated_dependency_files
 
       private
 
+      sig { returns(String) }
+      def job_file_path
+        ENV.fetch("DEPENDABOT_JOB_PATH")
```

Comment:
I see this is where we fetch this env var but I couldn't find where it actually gets written. Was it already being set before these changes? Other than that, changes look good to me 👍

Reply:
This is set by the CLI at the start of the job and will always be present.  In the few places we need it in a test I set it in a `before` block.